instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -388,13 +394,25 @@ def _start_response(status, response_headers, *args, **kwargs):
             duration_attrs_old = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.DEFAULT
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
```

Comment:
`HTTP_ROUTE` is not a defined attribute in the [old semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.11.0/specification/metrics/semantic_conventions/http-metrics.md#attributes) so this would just be a feature for the new sem conv.

Reply:
@lzchen sounds like you can answer https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1457#issuecomment-2179059109